src/main/java/core/basesyntax/service/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service.suppliers;
+
+import core.basesyntax.Circle;
+import core.basesyntax.Figure;
+import core.basesyntax.IsoscelesTrapezoid;
+import core.basesyntax.Rectangle;
+import core.basesyntax.RightTriangle;
+import core.basesyntax.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int figureIndex = new Random().nextInt(4);
+        switch (figureIndex) {
+            case 0:
+                return new Circle(colorSupplier.getRandomColor(), getRandomInt());
+            case 1:
+                return new IsoscelesTrapezoid(colorSupplier.getRandomColor(),
+                        getRandomInt(),
+                        getRandomInt(),
+                        getRandomInt());
+            case 2:
+                return new Rectangle(colorSupplier.getRandomColor(), getRandomInt(),
+                        getRandomInt());
+            case 3:
+                return new RightTriangle(colorSupplier.getRandomColor(), getRandomInt(),
+                        getRandomInt());
+            case 4:
+            default:
+                return new Square(colorSupplier.getRandomColor(),
+                        getRandomInt());
+        }
+    }
+
+    private static int getRandomInt() {
```

Comment:
```suggestion
    private int getRandomInt() {
```
Приватні методи повинні йти після публічних